api/envoy/api/v2/cluster/BUILD

```diff
@@ -33,3 +33,16 @@ api_go_proto_library(
     name = "outlier_detection",
     proto = ":outlier_detection",
 )
+
+api_proto_library_internal(
+    name = "filter",
+    srcs = ["filter.proto"],
+    visibility = [
+        "//envoy/api/v2:__pkg__",
+    ],
+)
+
+api_go_proto_library(
```

Comment:
I know this has been asked before, but can we please figure out a way to have this be built into `api_proto_library_internal`? You don't need to do this as part of this PR but can we either do that first or as an immediate follow up?

Reply:
I'm not an expert in Bazel + go, so not of much help here. There is a reason I gave up on maintaining bazel for gogoproto, and just use straight code generation. FWIW this is causing head ache in googleapis, and we ended up doing similar things of having per-language rules: https://github.com/googleapis/googleapis/blob/master/google/rpc/BUILD.bazel#L48